packages/server/lib/socket-e2e.ts

```diff
@@ -102,8 +102,7 @@ export class SocketE2E extends SocketBase {
         })
 
         socket.on('studio:init', (cb) => {
-          studio.getStudioModalShown()
-          .then(cb)
+          cb()
```

Comment:
If we aren't triggering anything, can we remove this event listener & remove the event emit from @packages/app?

Reply:
done